site/docs/layouts/shortcodes/dashboard.html

```diff
@@ -5,7 +5,8 @@
 			<th>Version</th>
 			<th>Life Stage</th>
 			<th>Design Stage</th>
-			<th>Verification Stage</th>
+            <th>Verification Stage</th>
+            <th>Commit ID</th>
```

Comment:
It looks like I copied in tabs when I originally copied the HTML for this from docgen.  I think it makes sense to just keep tabs here (or do another refactoring PR).